.github/workflows/system-tests.yml

```diff
@@ -324,3 +324,32 @@ jobs:
           checkState: ${{ needs.run-tests.result == 'skipped' && 'neutral' || needs.run-tests.result }}
           checkName: "System tests verification"
           checkDescription: "Check for overall system test verification"
+
+  perf-report:
+    name: Performance Report
+    needs:
+      - parse-params
+      - check-rights
+      - generate-matrix
+      - run-tests
+    # Only run if performance profile/pipeline was executed
+    if: |-
+      ${{
+        always() &&
+        needs.parse-params.outputs.shouldRun == 'true' &&
+        needs.check-rights.result == 'success' &&
+        github.event_name != 'pull_request' &&
+        needs.generate-matrix.outputs.isValid == 'true' &&
+        (contains(needs.parse-params.outputs.profileList, 'performance') || contains(needs.parse-params.outputs.pipelineList, 'performance'))
+      }}
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@v5
+        with:
+          ref: ${{ needs.parse-params.outputs.ref }}
+      - uses: ./.github/actions/systemtests/run-perf-report
+        with:
+          ref: ${{ needs.parse-params.outputs.ref }}
```

Comment:
is possible to add it to `report` job and just add condition to a specific step? Just asking...guess this is better anyway to have it a separated job